src/page-handlers/root-handler.ts

```diff
@@ -6,21 +6,18 @@ import { rootPageManager } from "../root-page-manager";
 export class RootPageHandler implements RestServer {
 
   async initializeRestServices(urlManager: UrlManager, app: Application): Promise<void> {
-    app.get('/', this.handleHomePage.bind(this));
-    app.get('/index.html', this.handleHomePage.bind(this));
-    app.get('/index.htm', this.handleHomePage.bind(this));
+    app.get('/', this.handleAppPage.bind(this));
```

Comment:
Doesn't /* handle everything now?  We don't need these other lines, do we?  (Except maybe / which might not match /*.)

Reply:
We dont need it. I just kept them just in case we need to offer different response later. They all point to the exact same method name. benign